marklogic-client-api/src/test/resources/dataservices/bulkOutputCallerNext.sjs

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const state  = fn.head(xdmp.fromJSON(endpointState));
+const work = fn.head(xdmp.fromJSON(workUnit));
+const limit = fn.head(xdmp.fromJSON(workUnit));
+
+const res = [];
+const d = fn.subsequence(fn.collection("bulkOutputCallerNext"), state.next, limit.max);
+res.push(state);
+for (const x of d) {
+res.push(x); };
+state.next = state.next + work.max;
+limit.max = limit.max + work.max;
```

Comment:
Is there a reason to increase the limit?

The limit is a constant that specifies the number of results to take after the offset.  

Also, because the work unit has constants and thus isn't returned from the endpoint, modifying the work unit doesn't change the next request.